src/components/Live/LiveEditor.js

```diff
@@ -6,21 +6,22 @@ import Editor from '../Editor';
 export default function LiveEditor(props) {
   return (
     <LiveContext.Consumer>
-      {({ code, onChange }) => (
+      {({ code, language, theme, onChange }) => (
         <Editor
-          {...props}
+          theme={theme}
```

Comment:
Do you need to explicitly pass these values in if they match the property name? I realize this might be a style pref so please feel free to disregard this comment 🙂

Reply:
I did it just to be explicit but yeah, it's kinda redundant!